src/Main.elm

```diff
@@ -175,13 +120,13 @@ view model =
                 (Element.text
                     (case model.status of
                         Failure ->
-                            Element.text "I was unable to load the text!"
+                            "I was unable to load the text!"
```

Comment:
This was close! You were passing `Element.text String` to `Element.text`, but it expects `String` only. 

Solution is to get rid of `Element.text` at either the top level (outside the `case`), or get rid of it from the individual case branches. I chose the latter in this case.